README.md

```diff
@@ -45,6 +45,7 @@ dictionary ContactInfo {
     sequence<DOMString> email;
     sequence<DOMString> tel;
     sequence<ContactAddress> address;
+    sequence<Blob> icon;
```

Comment:
Would it make sense for properties in this dictionary to match the `ContactProperty` order, i.e. alphabetized?

Reply:
Sure, I'll update it after we resolve the pending conversations